dvc/dependency/local.py

```diff
@@ -1,8 +1,53 @@
 from __future__ import unicode_literals
 
-from dvc.dependency.base import DependencyBase
+import os
+
+from dvc.pkg import Pkg
+
+from dvc.utils.compat import urlparse
+
+from dvc.dependency.base import DependencyBASE
 from dvc.output.local import OutputLOCAL
 
 
-class DependencyLOCAL(DependencyBase, OutputLOCAL):
-    pass
+class DependencyLOCAL(DependencyBASE, OutputLOCAL):
+    PARAM_PKG = "pkg"
+
+    def __init__(self, stage, info, remote=None):
+        pkg_info = info.pop(self.PARAM_PKG, None)
+
+        super(DependencyLOCAL, self).__init__(stage, info, remote=remote)
+
+        if not pkg_info:
+            return
+
+        self.pkg = Pkg(stage.repo.config, pkg_info)
+
+        if not self.pkg.installed:
```

Comment:
we actually should be throwing an exception ... otherwise it's not clear why do we need `dvc pkg install` in the first place

Reply:
anyway, it feels like an overkill that init of the Dependency is doing git clone and other stuff internally

Reply:
@shcheklein Then "one command" like `dvc pkg install https://github.com/user/proj data` won't work, you'll have to install first. Also, should install fetch data? If it should then, how much data? If it shouldn't, is it okay that dependency would do that? :slightly_smiling_face: For those reasons, I think that it is good practice to just install and fetch dynamically.